subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/ResolveConfigurationRepositoriesBuildOperationIntegrationTest.groovy

```diff
@@ -33,6 +33,7 @@ class ResolveConfigurationRepositoriesBuildOperationIntegrationTest extends Abst
     @Unroll
     def "repositories used when resolving project configurations are exposed via build operation (repo: #repo)"() {
         setup:
+        disablePluginRepoMirror()
```

Comment:
This test is checking the integrity of gradle plugin portal so we have to leave it as it is.